core/types/gen_header_json.go

```diff
@@ -11,32 +11,30 @@ import (
 	"github.com/dominant-strategies/go-quai/common/hexutil"
 )
 
-var _ = (*headerMarshaling)(nil)
-
 // MarshalJSON marshals as JSON.
 func (h Header) MarshalJSON() ([]byte, error) {
 	var enc struct {
 		ParentHash   					[]common.Hash   		`json:"parentHash"         			gencodec:"required"`
-		UncleHash    					common.Hash    			`json:"sha3Uncles"         			gencodec:"required"`
+		UncleHash    					common.Hash    			`json:"uncleHash"         			gencodec:"required"`
 		EVMRoot      					common.Hash   			`json:"evmRoot"            			gencodec:"required"`
 		QuaiStateSize                   *hexutil.Big            `json:"quaiStateSize"               gencodec:"required"`
 		UTXORoot		 				common.Hash	 			`json:"utxoRoot"           			gencodec:"required"`
 		TxHash       					common.Hash   			`json:"transactionsRoot"   			gencodec:"required"`
 		ReceiptHash  					common.Hash   			`json:"receiptsRoot"       			gencodec:"required"`
-		EtxHash      					common.Hash   			`json:"extTransactionsRoot"			gencodec:"required"`
+		EtxHash      					common.Hash   			`json:"etxsRoot"			gencodec:"required"`
```

Comment:
This should be OutboundEtxHash (because it's the root of etxs leaving the origin)

Reply:
changed it and also in the receipts